examples/portfolio/package.json

```diff
@@ -10,9 +10,6 @@
     "astro": "astro"
   },
   "dependencies": {
-    "preact": "^10.7.3",
-    "@astrojs/preact": "^1.0.2",
-    "astro": "^1.1.5",
-    "sass": "^1.52.2"
+    "astro": "^1.1.1"
```

Comment:
Just checking @Jutanium that you meant to include a lower version of Astro here that what was there before?

Reply:
Fixed, thanks!